lib/kwtool_web/templates/layout/app.html.eex

```diff
@@ -34,9 +37,34 @@
       </div>
     </nav>
     <main role="main" class="container">
-      <p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
-      <p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
-      <%= @inner_content %>
+      <%= render KwtoolWeb.SharedView, "_flash.html", conn: @conn %>
+      <div class="container">
+        <div class="row">
+          <nav class="sidebar">
+            <ul class="sidebar__nav">
+              <li class="sidebar__item">
+                <%= link "Dashboard", class: "sidebar__link", to: Routes.home_path(@conn, :index) %>
+              </li>
+              <li class="sidebar__item">
+                <a class="sidebar__link" href="#">
+                  Upload
+                </a>
+              </li>
+              <li class="sidebar__item">
+                <a class="sidebar__link" href="#">
+                  Report
+                </a>
+              </li>
+            </ul>
+          </nav>
+          <div class="content">
+            <%= @inner_content %>
+
+          </div>
+        </div>
+      </div>
+
+
```

Comment:
Should we remove these empty spaces 🧼  Also on line 62

Reply:
Updated